src/dotnet-scaffolding/dotnet-scaffold-aspire/ScaffoldSteps/FirstTimeTelemetrySetupStep.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.Core.Scaffolders;
+using Microsoft.DotNet.Scaffolding.Core.Steps;
+using Microsoft.DotNet.Scaffolding.Internal.Services;
+using Microsoft.DotNet.Scaffolding.Internal.Telemetry;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.Tools.Scaffold.Aspire.ScaffoldSteps;
+
+internal class FirstTimeTelemetrySetupStep : ScaffoldStep
```

Comment:
Could this (or whatever replaces it if we change it based on my above comment) be shared so you don't need it in each first party scaffolder?